### PR TITLE
corrections

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: LondonBritishLibrary/add/BLadd19341.xml [hunk 1]
```diff
@@ -69,7 +69,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
-                                    <material key="paper"/> wich bears <watermark>the date
+                                    <material key="paper"/> which bears <watermark>the date
```
Comment (eu-genia) [inline LondonBritishLibrary/add/BLadd19341.xml:72]: the visualization at the moment does not support this sort of encoding, 
"paper" is printed as a button, not part of text;
"watermark" is printed as yes/no; date is not parsed

If this needs to be fixed a separate issue would be needed

File: LondonBritishLibrary/add/BLadd19341.xml [hunk 2]
```diff
@@ -97,7 +97,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <desc>Neatly written, around <date>1810</date>.</desc>
                             </handNote>
                             <handNote xml:id="h2" script="Latin" corresp="#a1">
-                              <desc>Apparentely the handwritig of the traveller.</desc>
+                              <desc>Apparently the handwriting of the traveller.</desc>
                             </handNote>
                         </handDesc>
```
Comment (eu-genia) [inline LondonBritishLibrary/add/BLadd19341.xml:103]: to line 109 below: qhat is encoded as `q `I guess should be `note` (Extracts...)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/add/BLadd19341.xml (hunk 2 above)
Comment (thea-m) [reply LondonBritishLibrary/add/BLadd19341.xml:103]: No, I think it is fine like this: 
image

**Comment by thea-m** [comment]
Can this be merged?

**Review by eu-genia — APPROVED** [review]

**Pull request merged**